addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -962,3 +970,12 @@ section, .oe_img_bg, [data-oe-shape-data] {
         margin-bottom: 1rem;
     }
 }
+
+code.o_inline_code {
+    padding: .2em .4em;
+    border-radius: $border-radius;
+    border: 0;
```

Comment:
Is `border: 0;` still necessary in some context? In knowledge and website it doesn't seem to have a border by default